Misc/NEWS

```diff
@@ -381,6 +381,8 @@ Library
 - bpo-30508: Don't log exceptions if Task/Future "cancel()" method was
   called.
 
+- bpo-30645: use an internal variable while iterating in imp.load_package()
+
```

Comment:
"use an internal variable" is implementation detail and it's not suitable for NEWS entry.
Please describe the bug you fixed.